web/src/styles/scss/save-button.scss

```diff
@@ -0,0 +1,17 @@
+@import '@cmsgov/design-system-support/src/settings/index';
+
+.save-button--container .ds-l-col--3 {
+  margin-left: 601px;
+  text-align: right;
+}
+
+.save-button--container .ds-l-col--3 > div {
+  bottom: 0;
+  position: fixed;
+  width: 375px;
+  z-index: 10;
```

Comment:
Sets a width to make sure the alerts don't grow too big horizontally.  Sets a z-index so the alerts will float above textareas.